i18n/en/en.yaml

```diff
@@ -69,7 +69,7 @@ home_show_less:
 home_falco_what_is_falco_title:
   other: Detect security threats in real time
 home_falco_what_is_falco:
-  other: Falco is the open source solution for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into abnormal behaviors and potential security threats, intrusions, and data theft or compliance violations.
+  other: Falco is a cloud-native security tool designed for Linux systems. It employs custom rules on kernel events, which are enriched with container and Kubernetes metadata, to provide real-time alerts. Falco helps you gain visibility into abnormal behavior, potential security threats, and compliance violations, contributing to comprehensive runtime security.
```

Comment:
Note:
"off-host analysis" here might be unclear and potentially misleading, so I've removed it for now. Since we can't expand the message further here, we will work on a separate PR to explain it somewhere else on the homepage. @Issif is working on that :pray: